em_workflows/czi/flow.py

```diff
@@ -80,20 +80,18 @@ def gen_imageSet(file_path: FilePath) -> List:
             )
             assets.append(ng_asset)
         image_elt["assets"] = assets
-        imageSet.append(image_elt)
-    return imageSet
+        image_set.append(image_elt)
+    return image_set
 
 
-@task
-def generate_czi_imageset(file_path: FilePath):
+def bioformats_gen_zarr(file_path: FilePath):
```

Comment:
Turned into a separate function so that it can be mocked.
Mocked as in : if .zarr file has already been created, skip this function.